.github/workflows/python.yml

```diff
@@ -193,9 +193,7 @@ jobs:
             - name: Test pubsub with pytest
               working-directory: ./python
               run: |
-                  source .env/bin/activate
-                  pip install -r dev_requirements.txt
-                  pytest -v --asyncio-mode=auto -k test_pubsub --html=pytest_report.html --self-contained-html
+                  python3 dev.py test --args -k test_pubsub --html=pytest_report.html --self-contained-html
```

Comment:
@barshaul looks like you didn't change these in your other PR -- was that intentional?